advanced-achievements-api/src/main/java/com/hm/achievement/utils/PlayerAdvancedAchievementEvent.java

```diff
@@ -99,8 +99,8 @@ public String getCommandMessage() {
 		return commandRewards;
 	}
 
-	public ItemStack getItemReward() {
```

Comment:
This method is part of the public API. It should be deprecated rather than removed. I suggest an implementation along the following lines:
`return itemRewards == null || itemRewards .length == 0 ? null : itemRewards[0];`